website/src/views/JoinClass/JoinClass.jsx

```diff
@@ -145,26 +145,32 @@ class JoinClass extends Component {
         title: 'Class Code',
         dataIndex: 'name',
         key: 'name',
+        width: '15%',
```

Comment:
are these needed looks like this is causing tests needed

Reply:
Thanks I must of forgot about this i was testing something